hack/functests.sh

```diff
@@ -20,4 +20,4 @@ set -e
 
 source ./cluster/kubevirtci.sh
 
-go test ./tests --kubeconfig $(kubevirtci::kubeconfig) -ginkgo.v
+go test ./tests/... $E2E_TEST_ARGS --kubeconfig $(kubevirtci::kubeconfig) -ginkgo.v -test.v
```

Comment:
Why don't wee pass -v through E2E_TEST_ARGS? It should be up to the user to enable verbosity

Reply:
I think the verbose is important enough to be in the default. 
Would you prefer I removed it from here and added it as `E2E_TEST_ARGS="-ginkgo.v -test.v"` in the `functest` make recepie?

Reply:
The issue with verbose is that it spams the output with passing tests, but we are mostly interested in those that fails. So for CI run, dumping all is informative about progress at least. When running locally, I don't care about old passing tests.